### ISSUE TITLE
build system failure: DVTErrorPresenter: Unable to load simulator devices.

### ISSUE THREAD

**Issue by kelvin13** [SwiftPackageIndex/SwiftPackageIndex-Server#1670]
`swift-json` is getting some strange [build failures](https://swiftpackageindex.com/builds/4D50BEC5-46B9-4BF6-98FE-7BF2D3124486) on non-desktop devices (iOS, tvOS, watchOS). they don’t sound like they have to do with anything wrong in `swift-json`. is this a build system issue?

**Comment by kelvin13** [comment]
i don’t think this is related to #1299, because one of the failing builds is for `swift-json` 0.2.1 on iOS, for Swift 5.5, and that version of `swift-json` did not contain any submodules.

however, i will wait for a fresh build to run before reopening

**Comment by finestructure** [comment]
It's not submodules that are the issue here - it's the error `error: unable to resolve product type 'com.apple.product-type.tool' for platform 'iphoneos' (in target 'examples' from project 'swift-json')`. This is from the build failure you link at the top of this issue.

I'll update the issue title as well, as the `DVTErrorPresenter` warnings are a red herring. I seem to be getting these all the time when building with `xcodebuild` from the command line and our builder is no different, sadly.
